pulp_container/app/models.py

```diff
@@ -107,6 +118,73 @@ class Manifest(Content):
         through_fields=("image_manifest", "manifest_list"),
     )
 
+    def init_metadata(self, manifest_data=None, manifest_artifact=None, config_artifact=None):
```

Comment:
```suggestion
    def init_metadata(self, manifest_data=None):
```
These arguments are not used anywhere right now.